ProcessLib/CMakeLists.txt

```diff
@@ -37,4 +37,14 @@ endif()
 
 if(OGS_USE_PCH)
     cotire(ProcessLib)
+    if (OGS_USE_PYTHON)
+        # If Python is wrongly configured, in the Python headers the macro
+        # snprintf might be defined even though it's not needed. Precompiled headers
+        # cause that this macro definition is visible in all of ProcessLib.
+        # That might lead to clashes with jsoncpp. In particular the following
+        # compilation error was observed on MSVC:
```

Comment:
I could imagine that msvc version and python version would be helpful...

Reply:
Issue solved. Code and comment removed.